config/routes.rb

```diff
@@ -4,7 +4,7 @@
   devise_for :admins
   resources :reservations
 
-  root "/postings#splash"
+  root "welcome#welcome"
```

Comment:
Take out the "welcome#welcome"  and use:  root '/posting#splash'  that's what Ryan is coding to.